website/pages/en/querying/graphql-api.mdx

```diff
@@ -376,9 +389,9 @@ Graph Node implements [specification-based](https://spec.graphql.org/October2021
 
 ## Schema
 
-The schema of your data source--that is, the entity types, values, and relationships that are available to query--are defined through the [GraphQL Interface Definition Langauge (IDL)](https://facebook.github.io/graphql/draft/#sec-Type-System).
+The schema of your `data source`, which is the entity types, values, and relationships that are available to query, are defined through the [GraphQL Interface Definition Langauge (IDL)](https://facebook.github.io/graphql/draft/#sec-Type-System).
 
-GraphQL schemas generally define root types for `queries`, `subscriptions` and `mutations`. The Graph only supports `queries`. The root `Query` type for your subgraph is automatically generated from the GraphQL schema that's included in your subgraph manifest.
+GraphQL schemas generally define root types for `queries`, `subscriptions` and `mutations`. The Graph only supports `queries`. The root `Query` type for your subgraph is automatically generated from the GraphQL schema that's included in your [subgraph manifest](/developing/creating-a-subgraph/#components-of-a-subgraph).
 
 > **Note:** Our API does not expose mutations because developers are expected to issue transactions directly against the underlying blockchain from their applications.
```

Comment:
```suggestion
> Note: Our API does not expose mutations because developers are expected to issue transactions directly against the underlying blockchain from their applications.
```